lib/builderator/config/defaults.rb

```diff
@@ -80,6 +80,16 @@ module Config
         end
       end
 
+      profile :docker do |profile|
+        profile.log_level :info
+
+        profile.packer do |packer|
+          packer.build :docker do |build|
+            build.type 'docker'
```

Comment:
Not that I can see. All other attributes either have Packer defaults or should be specified in the Buildfile.